tests/src/test_cases/libs/test_font_stress.c

```diff
@@ -0,0 +1,208 @@
+#if LV_BUILD_TEST
+#include "../lvgl.h"
+#include "rnd_unicodes/lv_rnd_unicodes.h"
```

Comment:
I think these includes should go inside `#if LV_USE_FREETYPE`, because if FreeType library is disabled, then we don't use these.

Reply:
done